_layouts/sidebar.md

```diff
@@ -5,6 +5,10 @@
 * Resources
   - [DevOps](./../pages/devops-resources.md)
   - [Web Development](./../pages/web-development-resources.md)
+    - [HTML resources](./../pages/HTML-resources.md)
+    - [CSS resources](./../pages/css-resources.md)
+    - [react](./../pages/react-resources.md)
```

Comment:
Hey @elidakirigo, please capitalize the 'react' to 'React'

Reply:
ok

> Hey @elidakirigo, please capitalize the 'react' to 'React'

Reply:
i have